docs/src/content/docs/fr/resources/plugins.mdx

```diff
@@ -78,6 +78,18 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
 		title="starlight-versions"
 		description="Versionner vos pages de documentation Starlight."
 	/>
+	<LinkCard
+		href="https://inox-tools.fryuni.dev/star-warp"
+		title="star-warp"
+		description="Accéder aux résultats de recherche de votre documentation à la vitesse de la lumière."
```

Comment:
References as it's a bit of a tricky one:

- https://en.wikipedia.org/wiki/Warp_drive
- https://fr.wikipedia.org/wiki/Distorsion_(Star_Trek)
- https://www.reddit.com/r/etymology/comments/10w3s6m/when_did_warp_first_start_being_synonymous_with/